examples/tictactoe/ts/src/tictactoe.ts

```diff
@@ -191,26 +195,26 @@ export class TicTacToe extends SmartContract {
     const player = pubkey.equals(player2); // player 1 is false, player 2 is true
 
     // ensure its their turn
-    const nextPlayer = this.nextPlayer.get();
-    this.nextPlayer.assertEquals(nextPlayer); // precondition that links this.nextPlayer.get() to the actual on-chain state
+    const nextPlayer = this.nextIsPlayer2.get();
+    this.nextIsPlayer2.assertEquals(nextPlayer); // precondition that links this.nextPlayer.get() to the actual on-chain state
```

Comment:
update comment to say `precondition that links this.nextPlayer2.get() to the actual on-chain state`